src/PKSim.Core/Commands/EditParameterCommand.cs

```diff
@@ -51,36 +54,40 @@ protected void UpdateDependenciesOnParameter(IExecutionContext context)
       private void udpateBuildingBlockProperties(IExecutionContext context)
       {
          var parentBuildingBlock = context.BuildingBlockContaining(_parameter) ?? context.Get<IPKSimBuildingBlock>(BuildingBlockId);
-         context.UpdateBuildinBlockProperties(this, parentBuildingBlock);
+         context.UpdateBuildinBlockPropertiesInCommand(this, parentBuildingBlock);
       }
 
       protected abstract void UpdateParameter(IParameter parameter, IExecutionContext context);
 
       protected IParameter OriginParameterFor(IParameter parameter, IExecutionContext context)
       {
-         if (!IsBuildingBlockParameterInSimulation(parameter))
+         if (!isBuildingBlockParameterInSimulation(parameter))
             return null;
 
          return context.Get<IParameter>(parameter.Origin.ParameterId);
       }
 
-      protected bool IsBuildingBlockParameterInSimulation(IParameter parameter)
+      private bool isBuildingBlockParameterInSimulation(IParameter parameter)
```

Comment:
Looking at this logic, I simply reorganized the if statements and updated the comments that were ...well outdated as always :)